examples/multiphase_flow/ex0/LSLocateCircularInterface.cpp

```diff
@@ -82,7 +82,7 @@ LSLocateCircularInterface::setLevelSetPatchData(int D_idx,
                 IBTK::Vector coord = IBTK::Vector::Zero();
                 Pointer<CartesianPatchGeometry<NDIM> > patch_geom = patch->getPatchGeometry();
                 const double* patch_X_lower = patch_geom->getXLower();
-                const hier::Index<NDIM>& patch_lower_idx = patch_box.lower();
+                const Index<NDIM>& patch_lower_idx = patch_box.lower();
```

Comment:
This needs to be qualified by `hier::`.

Reply:
(Please check that `Index` is not use unqualified in other places in the new examples --- this is something that breaks some compilers but not others.)